irc3/compat.py

```diff
@@ -13,9 +14,36 @@
 
 try:  # pragma: no cover
     import asyncio
-    from asyncio.queues import Queue
+    from asyncio.queues import Queue as BaseQueue
     from asyncio.queues import QueueFull  # NOQA
+    from asyncio import sleep as _original_sleep
+    from asyncio import wait as _original_wait
 except ImportError:  # pragma: no cover
     import trollius as asyncio  # NOQA
-    from trollius.queues import Queue  # NOQA
+    from trollius.queues import Queue  as BaseQueue # NOQA
```

Comment:
pretty sure trollius can be removed. this was required for py2, maybe py3.3 max (?)

python3.5 should not require this

Reply:
Perhaps that cleanup best happens in a separate PR?

Reply:
I don't really care :) I can merge as this if it's good enough for you

Reply:
Sounds good to me